### PR TITLE
add pippel

### PR DESCRIPTION
**Pull request by brotzeitmacher**
### Brief summary of what the package does

Emacs frontend to python package manager pip.

### Direct link to the package repository

https://github.com/brotzeitmacher/pippel

### Your association with the package

maintainer

### Relevant communications with the upstream package maintainer

None needed

### Checklist

Please confirm with `x`:

- [x] I've read [CONTRIBUTING.md](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.md)
- [x] I've used [flycheck](https://github.com/purcell/package-lint) to check for packaging issues, and addressed its feedback
- [x] I've built and installed the package using the instructions in the [README](https://github.com/melpa/melpa/blob/master/README.md)

### CODE REVIEW

**Comment by syohex** [comment]
- I have sent PR(https://github.com/brotzeitmacher/pippel/pull/1)
  - Some features requires Emacs 25 or higher versions
  - Miss package dependency
  - etc

**Comment by brotzeitmacher** [comment]
Merged it. Thanks.

**Comment by syohex** [comment]
- There is no autoload cookie

File: recipes/pippel [hunk 1]
```diff
@@ -0,0 +1,5 @@
+(pippel
+ :fetcher github
+ :repo "brotzeitmacher/pippel"
+ :files ("*.el"
```
Comment (syohex) [inline recipes/pippel]: Please use `:defaults` instead of `*.el`.

**Comment by brotzeitmacher** [comment]
Added autoload cookies and fixed recipe.

**Comment by syohex** [comment]
Merged

**Pull request merged**